test/basictests.jl

```diff
@@ -27,20 +27,20 @@ function test_interface(alg, prob1, prob2)
     b2 = prob2.b
     x2 = prob2.u0
 
-    y = solve(prob1, alg; cache_kwargs...)
-    @test A1 * y ≈ b1
+    sol = solve(prob1, alg; cache_kwargs...)
+    @test A1 * sol.u ≈ b1
 
     cache = SciMLBase.init(prob1, alg; cache_kwargs...) # initialize cache
-    y = solve(cache)
-    @test A1 * y ≈ b1
+    sol = solve(cache)
+    @test A1 * sol.u ≈ b1
 
-    cache = LinearSolve.set_A(cache, copy(A2))
-    y = solve(cache; cache_kwargs...)
-    @test A2 * y ≈ b1
+    cache = LinearSolve.set_A(cache, deepcopy(A2))
```

Comment:
why is this necessary?

Reply:
because `Base.copy` is not defined for some `SciMLOperators` like `FunctionOperator`.